src/commands/config.command.ts

```diff
@@ -0,0 +1,63 @@
+import { Command, ensureFile, Input, prompt } from "../deps.ts";
+import dir from "https://deno.land/x/dir@1.5.1/mod.ts";
+import { fileExists, isNotEmpty, runCli } from "../helpers/index.ts";
+
+export const configCommand = new Command()
+  .description("contribution-mate configuration")
+  .action(async () => {
```

Comment:
suggestion(blocking): action and declaration should be separated
We could create a folder structure like so:
`src/commands/config/config.ts` - action declaration
`src/commands/config/index.ts` - command declaration and export